### PR TITLE
German Stringtable

### PR DESCRIPTION
**Pull request by nudelchef**
Item_Slayer_description says that it increases attack speed, but it doesn't. This commit changes that in the english and german stringtable.

This commit also updates the german stringtable with a LOT of changes. 😄

### CODE REVIEW

File: game/stringtables/entities_de.str [hunk 1]
```diff
@@ -493,12 +493,12 @@ Ability_Vex1_description_simple                                            Schwi
...
-Ability_Vex2_description                                                   Aktiviere um drei umliegenden Feinden  ^o$damage$ Magieschaden^* in 6 Sekunden anzurichten. Kann bis zu 70% Bonus-Schaden anrichten, wenn es ein einziges Ziel gibt.
+Ability_Vex2_description                                                   Aktiviere um drei umliegenden Feinden ^o$damage$ Magieschaden^* in 6 Sekunden anzurichten. Kann bis zu 70% Bonus-Schaden anrichten, wenn es ein einziges Ziel gibt.
 Ability_Vex2_description_simple                                            fügt umliegenden Feinden für einige Zeit Schaden zu
 
 // Ability_Vex3
 Ability_Vex3_name                                                          Macht-Nichts-Schild
-Ability_Vex3_description									Markiere einen Verbündeten, um alle Betäubungs-/Lähmeffekte zu beseitigen und einen ^o$shield$ Schild^* zu gewähren, der 2 Sekunden lang 70 % Widerstand gegen Betäubungs-/Lähmeffekte verleiht.
+Ability_Vex3_description									               Markiere einen Verbündeten, um alle Betäubungs-/Lähmeffekte zu beseitigen und einen ^o$shield$ Schild^* zu gewähren, der 2 Sekunden lang 70 % Widerstand gegen Betäubungs-/Lähmeffekte verleiht.
```
Comment (padjon) [inline game/stringtables/entities_de.str:501]: 70<space>%
Remove spaces consequently in %-values

File: game/stringtables/entities_de.str [hunk 2]
```diff
@@ -733,7 +733,7 @@ Hero_Caprice_lore                                                          Capri
 
 // Ability_CapriceTutorial1
 Ability_CapriceTutorial1_name                                              Feuerlager
-Ability_CapriceTutorial1_description_simple								Wähle eine Richtung, um ^o$damage$ Zauberschaden^* in einer Linie auszuüben.  Nach einer kurzen Verzögerung explodiert diese Linie und übt den halben Schaden aus.
+Ability_CapriceTutorial1_description_simple								   Wähle eine Richtung, um ^o$damage$ Zauberschaden^* in einer Linie auszuüben.  Nach einer kurzen Verzögerung explodiert diese Linie und übt den halben Schaden aus.
```
Comment (padjon) [inline game/stringtables/entities_de.str:736]: Doube whitespace  before "  Nach einer kurzen..."

**Review by padjon — CHANGES_REQUESTED** [review]
A few double whitespaces and multiple times percentage values with whitespaces should get xleaned up for consistency - liike 90 % to 90%

File: game/stringtables/entities_de.str [hunk 3]
```diff
@@ -1,214 +1,235 @@
...
 EffectType_StatusDisable
 EffectType_StatusStealth
-EffectType_Push                                                            Offensive
-EffectType_Splash                                                          Flächenschaden
-EffectType_DOT                                                             Zeitschaden
-EffectType_Buff                                                            Buffs
+EffectType_Push														Offensive
+EffectType_Splash													Flächenschaden
+EffectType_DOT
+EffectType_Buff
```
Comment (padjon) [inline game/stringtables/entities_de.str:16]: Why was the value removed here? Am i missing something? Same for _DOT

**Review by padjon — COMMENTED** [review]
Requires clarification

File: game/stringtables/entities_de.str [hunk 4]
```diff
@@ -1,214 +1,235 @@
...
+EffectType_AttackDamage												Angriffsschaden
+EffectType_Magic													Magie
+EffectType_StatusBuff												Buff
+EffectType_StatusDebuff												Debuff
 EffectType_StatusDisable
 EffectType_StatusStealth
-EffectType_Push                                                            Offensive
-EffectType_Splash                                                          Flächenschaden
-EffectType_DOT                                                             Zeitschaden
-EffectType_Buff                                                            Buffs
```
Comment (padjon) [inline game/stringtables/entities_de.str:17]: For clarification, this was the previous value

**Review by padjon — APPROVED** [review]

File: game/stringtables/entities_de.str (hunk 3 above)
Comment (nudelchef) [reply game/stringtables/entities_de.str:16]: I've used a self written python script to merge the english and german file. 
I didn't account for the fact, that the german file might have a value, that the english one has not set at all.
The translation for those two are correct, as they were originally.

**Comment by padjon** [comment]
Thanks for clarification. Great work, great PR.

**Pull request merged**